uaa/src/test/java/org/cloudfoundry/identity/uaa/jwt/degraded/JwtBearerGrantDegradedIntegrationTest.java

```diff
@@ -45,18 +47,40 @@
     private static final String TENANT_ID = "t10";
     private final static String DEVICE_ID = "d10";
     private final static String DEVICE_CLIENT_ID = "c1";
-    private static final String AUDIENCE = "http://localhost:8080/uaa/oauth/token";
 
     protected final static Logger logger = LoggerFactory.getLogger(JwtBearerGrantDegradedIntegrationTest.class);
 
 
     @Value("${integration.test.base_url}")
     private String baseUrl;
 
+    @Value("${PUBLISHED_HOST:predix-uaa-integration}")
+    String publishedHost;
+
+    @Value("${CF_DOMAIN:run.aws-usw02-dev.ice.predix.io}")
+    String cfDomain;
+
+    @Value("${BASIC_AUTH_CLIENT_ID:app}")
+    String basicAuthClientId;
+
+    @Value("${BASIC_AUTH_CLIENT_SECRET:appclientsecret}")
+    String basicAuthClientSecret;
```

Comment:
Remove this secret default ... in case an error happens with tests and the env is not torn down.